src/test/daml/Daml/Finance/Data/Test/DateClock.daml

```diff
@@ -0,0 +1,63 @@
+-- Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Daml.Finance.Data.Test.DateClock where
+
+import DA.Assert ((===))
+import DA.Date (addDays, toDateUTC)
+import DA.Set qualified as S (singleton)
+import Daml.Finance.Data.Time.DateClock qualified as DateClock (DateClock(..), T)
+import Daml.Finance.Data.Time.DateClock.Types (Unit(..))
+import Daml.Finance.Interface.Data.Reference.Time qualified as Time (Advance(..), I, Rewind(..))
+import Daml.Finance.Interface.Types.Common.Types (Id(..))
+import Daml.Script
+
+
+testDateClock : Script ()
+testDateClock = script do
+  bank <- listKnownParties
```

Comment:
We don't expect these tests to be run against a running ledger, so I'd just use `allocatePartyWithHint` here

Reply:
Dropped.